python/lsst/ts/salobj/topics/read_topic.py

```diff
@@ -215,35 +215,36 @@ class ReadTopic(BaseTopic):
     -----
     **Queues**
 
-    There are actually two queues: a Python queue whose length
-    is set by ``queue_len`` and a dds queue whose length is set by
-    the DDS QoS file. Data can be lost in two ways:
+    There are two queues: a Python queue whose length is set by ``queue_len``
```

Comment:
I think a sentence or two on why both exist might be helpful?
"The Python queue exists to enable ..."

Reply:
I pushed another change with that info.

Reply:
Good suggestion. I added this as another commit. I would love to just use the dds queue but between the severely limited API and lack of asyncio support it did not seem practical.